src/js/button.js

```diff
@@ -1,7 +1,9 @@
 import Component from './component';
-import * as Lib from './lib';
-import * as Events from './events';
+import * as Dom from './utils/dom.js';
+import * as Events from './utils/events.js';
+import * as Fn from './utils/fn.js';
 import document from 'global/document';
+import assign from 'object.assign';
```

Comment:
That'd be cool. I wonder though if it's even important that we use the native one ever, since we're including the module either way. I guess there could be performance benefits, but it also creates two different code paths that could (while unlikely) cause differences between browsers.